packages/ast/src/encoding/__tests__/__snapshots__/authz.timestamp.test.ts.snap

```diff
@@ -0,0 +1,227 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`cosmos/authz/v1beta1/authz date amino 1`] = `
+"export const AminoConverter = {
+  \\"/cosmos.authz.v1beta1.Grant\\": {
+    aminoType: \\"cosmos-sdk/Grant\\",
+    toAmino: ({
+      authorization,
+      expiration
+    }: Grant): AminoGrant[\\"value\\"] => {
+      return {
+        authorization: {
+          type_url: authorization.typeUrl,
+          value: authorization.value
+        },
+        expiration
+      };
+    },
+    fromAmino: ({
+      authorization,
+      expiration
+    }: AminoGrant[\\"value\\"]): Grant => {
+      return {
+        authorization: {
+          typeUrl: authorization.type_url,
+          value: authorization.value
```

Comment:
will be fixing this by parsing any fields in a future branch https://github.com/osmosis-labs/telescope/issues/118
```proto
  google.protobuf.Any authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
```